### DISCUSSION TITLE
getting compile error for amazon-ivs-wasmworker.min.wasm

### DISCUSSION THREAD

**Discussion by sathia-s** [quasarframework/quasar#10492] [General - Components / Directives / etc]
Hi, can someone here help to guide how I can compile the amazon-ivs-wasmworker.min.js amazon-ivs-wasmworker.min.wasm which is placed under the public directory. 

errors:
amazon-ivs-wasmworker.min.js:5 wasm streaming compile failed: TypeError: Failed to execute 'compile' on 'WebAssembly': Incorrect response MIME type. Expected 'application/wasm'.
printErr @ amazon-ivs-wasmworker.min.js:5
amazon-ivs-wasmworker.min.js:5 falling back to ArrayBuffer instantiation
printErr @ amazon-ivs-wasmworker.min.js:5
amazon-ivs-wasmworker.min.js:5 failed to asynchronously prepare wasm: CompileError: WebAssembly.instantiate(): expected magic word 00 61 73 6d, found 2f 2a 21 0a @+0
printErr @ amazon-ivs-wasmworker.min.js:5
amazon-ivs-wasmworker.min.js:5 CompileError: WebAssembly.instantiate(): expected magic word 00 61 73 6d, found 2f 2a 21 0a @+0
onAbort @ amazon-ivs-wasmworker.min.js:5
amazon-ivs-wasmworker.min.js:5 CompileError: WebAssembly.instantiate(): expected magic word 00 61 73 6d, found 2f 2a 21 0a @+0
printErr @ amazon-ivs-wasmworker.min.js:5
amazon-ivs-wasmworker.min.js:5 Uncaught (in promise) RuntimeError: abort(CompileError: WebAssembly.instantiate(): expected magic word 00 61 73 6d, found 2f 2a 21 0a @+0). Build with -s ASSERTIONS=1 for more info.
    at z (amazon-ivs-wasmworker.min.js:5)
    at amazon-ivs-wasmworker.min.js:5

I managed to get both files work in ionic out of the box.

**Comment by ontwikkelfabriek** [comment]
@sathia-s 

Create a codepen.io or another demo setup so we can provide better support.
